docs/index.md

```diff
@@ -46,9 +46,12 @@ cd LibreLingo
 
 ### Web app
 
+You will need [Node](https://nodejs.org/en/). Note that this project is not yet compatible with Node v16.
```

Comment:
How come it's not compatible with node 16 yet?

Reply:
Because it uses [node-sass v5](https://github.com/kantord/LibreLingo/blob/0448295e71fd2e0b8b6190f067c90a0e80586f3b/apps/web/package.json#L99).

[node-sass v6 is the first to support node 16.](https://github.com/sass/node-sass#node-version-support-policy)